clients/cloud/scala/src/main/resources/ccloud.properties

```diff
@@ -0,0 +1,7 @@
+ssl.endpoint.identification.algorithm=https
```

Comment:
This config file specifies all the properties, while the other examples only put the user-specific properties in their config file (bootstrap, sasl user/pass) and hard-code the other ones in the code.

I think there is value to being consistent, but I don't have a strong opinion in the java/scala case.